ls1intum/Artemis
# `General`: Improve performance and appearance

### Checklist
#### General



### Motivation and Context
1. Make the for-dashboard call faster by reducing the number of database calls
2. Improves the styling of the course overview and the course management view

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise with Complaints enabled

1. Open the course overview and the course management view and check that everything looks good
2. Navigate into the course and several sub pages and make sure everything looks ok



### Review Progress






#### Code Review
#### Manual Tests
